cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -52,6 +53,16 @@ export KUBEVIRTCI_GOCLI_CONTAINER=quay.io/kubevirtci/gocli:latest
     ${ssh} node02 -- ip l show eth1
     ${ssh} node02 -- ip l show eth2
 
+    # Verify Multus v3 image is used
+    readonly expected_multus_v3_image="ghcr.io/k8snetworkplumbingwg/multus-cni:v3.8"
+    actual_multus_image=$(${ksh} get ds -n kube-system kube-multus-ds -o jsonpath='{.spec.template.spec.containers[0].image}')
+    [ "${actual_multus_image}" == "${expected_multus_v3_image}" ]
```

Comment:
Let's just simplify this and skip checking the "minor" part of the semver
```suggestion
    ${ksh} get ds -n kube-system kube-multus-ds -o yaml |grep multus-cni:v3
```

Reply:
done